src/data/markdown/docs/30 k6-browser/02 API/03 BrowserType/connect--options--.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'connect(wsURL, [options])'
+excerpt: 'Browser module: BrowserType.connect method'
+---
+
+Connects to an existing browser instance.
+
+| Parameter         | Type     | Default | Description                                                                                                                                                                                                                                                           |
+|-------------------|----------|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| debug             | boolean  | `false` | All CDP messages and internal fine grained logs will be logged if set to `true`.                                                                                                                                                                                      |
+| slowMo            | string   | `null`  | Slow down input actions and navigation by the specified time e.g. `'500ms'`.                                                                                                                                                                                          |
+| timeout           | string   | `'30s'` | Default timeout to use for various actions and navigation.                                                                                                                                                                                                            |
+
+
+### Returns
+
+| Type   | Description                                            |
+|--------|--------------------------------------------------------|
+| object | [Browser](/javascript-api/k6-browser/api/browser/) object |
+
+
+## Example
+
+<CodeGroup labels={[]}>
+
+```javascript
+import { chromium } from 'k6/experimental/browser';
+
+export default async function () {
+  const wsURL = 'ws://browser.farm.example.com:9222/devtools/browser/a7ee4f2d-35cf-4478-a333-f597e1532ab0';
+  const browser = chromium.connect(wsURL, {
+    debug: true,
+    slowMo: '500ms',
+    timeout: '30s',
+  });
+  const context = browser.newContext();
+  const page = context.newPage();
+
+  try {
+    await page.goto('http://whatsmyuseragent.org/', { waitUntil: 'networkidle' });
```

Comment:
I think in our examples we should try to work with `test.k6.io` or another domain that we control and don't mind getting hammered.

```suggestion
    await page.goto('https://test.k6.io/', { waitUntil: 'networkidle' });
```

Reply:
Personally I agree with you, I used this site because it was the one used in the example for `launch`, which is the other alternative to create/attach to a browser. See [here](https://k6.io/docs/javascript-api/k6-browser/api/browsertype/launch/#example). But I think using `test.k6.io` would make more sense. I will change it.

Reply:
Changed in [b1fa39e](https://github.com/grafana/k6-docs/pull/1114/commits/b1fa39e0cae20adcd24571273da838c9f194ee0e).